CRM/Contact/BAO/Query.php

```diff
@@ -2867,7 +2867,7 @@ public function includeContactSubTypes($value, $grouping, $op = 'LIKE') {
     }
     elseif (is_array($value)) {
       foreach ($value as $k => $v) {
-        if (!empty($k)) {
+        if (isset($k)) {
```

Comment:
This doesn't make any sense to me, but neither did it previously. Should we just remove the condition altogether?